src/tests/CMakeLists.txt

```diff
@@ -113,7 +113,7 @@ if (${io_tests})
     can_build_epoll_io_tests(epoll_tests)
     if(${epoll_tests})
         add_test(NAME LogDev-Epoll COMMAND ${CMAKE_BINARY_DIR}/bin/test_log_dev)
-        add_test(NAME LogStore-Epoll COMMAND ${CMAKE_BINARY_DIR}/bin/test_log_store)
+        #add_test(NAME LogStore-Epoll COMMAND ${CMAKE_BINARY_DIR}/bin/test_log_store)
```

Comment:
hanging in github CI, seen also in other PR's CI build: https://github.com/eBay/HomeStore/actions/runs/9107660378/job/25036931542?pr=426

Reply:
https://github.com/eBay/HomeStore/issues/424 is tracking this